packages/expo-image-picker/android/src/main/java/expo/modules/imagepicker/ImagePickerModule.kt

```diff
@@ -221,7 +245,58 @@ class ImagePickerModule(private val mContext: Context,
       setOutputCompressQuality(pickerOptions.quality)
     }
 
-    startActivityOnResult(cropImageBuilder.getIntent(context), CropImage.CROP_IMAGE_ACTIVITY_REQUEST_CODE, promise)
+    startActivityOnResult(cropImageBuilder.getIntent(context), CropImage.CROP_IMAGE_ACTIVITY_REQUEST_CODE, promise, pickerOptions)
+  }
+
+  //endregion
+
+  // region ActivityEventListener
```

Comment:
Is it possible to delegate this whole region to separate class?

Reply:
I can do it but in a different PR ;)